core/src/main/java/nu/yona/server/rest/RestClientErrorHandler.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Stichting Yona Foundation
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.rest;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.web.client.ResponseErrorHandler;
+
+public class RestClientErrorHandler implements ResponseErrorHandler
+{
+	private static final Logger logger = LoggerFactory.getLogger(RestClientErrorHandler.class);
+
+	@Override
+	public void handleError(ClientHttpResponse response) throws IOException
+	{
+		logger.error("Response error: {} {}", response.getStatusCode(), response.getStatusText());
+	}
+
+	@Override
+	public boolean hasError(ClientHttpResponse response) throws IOException
```

Comment:
It's one of the methods of the implemented interface. Explaining that in a comment sounds like overkill to me.